x-pack/plugin/ilm/qa/multi-node/src/test/java/org/elasticsearch/xpack/indexlifecycle/TimeSeriesLifecycleActionsIT.java

```diff
@@ -151,8 +151,8 @@ public void testMoveToRolloverStep() throws Exception {
             "}");
         client().performRequest(moveToStepRequest);
         assertBusy(() -> assertTrue(indexExists(secondIndex)));
-        assertBusy(() -> assertFalse(indexExists(shrunkenOriginalIndex)));
         assertBusy(() -> assertFalse(indexExists(originalIndex)));
+        assertBusy(() -> assertFalse(indexExists(shrunkenOriginalIndex)));
```

Comment:
If the order of these asserts is important in a non-obvious way, we should probably include a comment explaining why.